lib/fastlane/actions/lcov.rb

```diff
@@ -0,0 +1,83 @@
+module Fastlane
+  module Actions
+    class LcovAction < Action
+      @@derived_data_path = "#{File.expand_path('~')}/Library/Developer/Xcode/DerivedData/"
```

Comment:
I don't think that's the Ruby way to do this. Could you refactor this maybe into its own methods?